### PR TITLE
Orders service

### PR DESCRIPTION
**Pull request by PaulFenton**
Added basic API functionality to support the dropdowns and data selections for the Order and Shipping UI's

### CODE REVIEW

File: A2-2015/A2-Rewrite/backend/index.js [hunk 1]
```diff
@@ -33,32 +33,79 @@ const Product = sequelize.define('product', {
...
+        customer_last_name: Sequelize.STRING,
+        customer_address: Sequelize.STRING,
+        customer_phone: Sequelize.STRING,
+        price: Sequelize.DOUBLE,
+        shipped_flag: Sequelize.BOOLEAN
+    },{
+        underscored: true,
+        freezeTableName: true,
+        tableName: 'order',
+        timestamps: false
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/backend/index.js:83]: Why not leveraging sequelize timestamps?

File: A2-2015/A2-Rewrite/backend/index.js [hunk 2]
```diff
@@ -33,32 +33,79 @@ const Product = sequelize.define('product', {
...
+});
+
+const OrderItems = sequelize.define('order_item', {
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/backend/index.js:86]: I think we should add the price too right?

File: A2-2015/A2-Rewrite/backend/index.js [hunk 3]
```diff
@@ -114,8 +161,52 @@ app.post('/auth/log/:type',jsonParser,async function (req, res) {
...
+        res.json(await Product.findAll({}));
+    }
+});
+
+app.get('/product_categories', async function(req, res) {
+    res.json(await ProductCategory.findAll({}));
+});
+
+app.get('/orders', async  function (req, res) {
+    var showPending = ('false' != req.param('show_pending', true));
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/backend/index.js:182]: 👍

File: README.md [hunk 4]
```diff
@@ -43,8 +43,8 @@ nvm install 8.10.0
 #### 4. set backend database credentials
 
 ```bash
-export EEP_DATABASE_ADMIN_NAME='eep_admin'
-export EEP_DATABASE_PASSWORD='eep_password'
+export EEP_DATABASE_ADMIN_NAME=<username from email>
```
Comment (darioalessandro) [inline README.md:46]: 👍

File: README.md [hunk 5]
```diff
@@ -100,6 +100,18 @@ you should see the UI
 
 ![GitHub Logo](A2-2015/A2-Rewrite/imgs/menu.gif)
 
+### Using the Back-End API (Examples)
```
Comment (darioalessandro) [inline README.md:103]: 👍 @kayvanhaverbeck these are the services.

**Review by darioalessandro — APPROVED** [review]
LGTM 

why did you choose to turn off sequelize timestamps?

File: A2-2015/A2-Rewrite/backend/index.js (hunk 1 above)
Comment (PaulFenton) [reply A2-2015/A2-Rewrite/backend/index.js:83]: we can.. I just didn't put them in the schema originally as I don't know how they would be used.. should I add?

File: A2-2015/A2-Rewrite/backend/index.js (hunk 2 above)
Comment (PaulFenton) [reply A2-2015/A2-Rewrite/backend/index.js:86]: I think the price lives with in the product table, unless we want to provide the possibility of price being different for different orders.. ideally should only store data once and then do aggregation / joins whenever we need to report thing

**Pull request merged**